chart/values.yaml

```
@@ -13,6 +13,8 @@ image:
   repo: openebs
   # -- Release tag for our images
   tag: develop
+  # -- Chart
+  chart: mayastor
```

```suggestion
  prefix: mayastor
```

Do we want this to just be the prefix, or do we want this to be used for other things?

---

done with the changes.